src/justbases/_constants.py

```diff
@@ -38,7 +38,7 @@ def __str__(self):
         return self.__class__.__name__
 
     def __repr__(self):
-        return "%s(%s)" % (str(self), self.doc)
+        return f"{str(self)}({self.doc})"
```

Comment:
```str``` call ought to be unnecessary, even if it was there before.

Reply:
I removed the `str()` call from `str(self)`; does that work?

Reply:
Should be good.